src/mastercore.h

```diff
@@ -130,6 +130,10 @@ enum FILETYPES {
 #define OMNI_PROPERTY_MSC   1
 #define OMNI_PROPERTY_TMSC  2
 
+#include <boost/multiprecision/cpp_dec_float.hpp>
```

Comment:
Why is this and the following two lines here? `cpp_dec_float_100` and neither `XDOUBLE` are used in the header, as far as I can see.